python/tk_multi_workfiles/work_files.py

```diff
@@ -125,10 +125,21 @@ def wrapper(*args, **kwargs):
         return res
     return wrapper
 
+
 class WorkFiles(object):
     """
     Main entry point for all commands in the app.
     """
+
+    def __init__(self):
+        """
+        Constructor.
+        """
+        app = sgtk.platform.current_bundle()
```

Comment:
This will sync the path cache every single time the file manager (open and save) is opened.